test/common/upstream/health_checker_impl_test.cc

```diff
@@ -2444,9 +2444,10 @@ class GrpcHealthCheckerImplTestBase {
         });
   }
 
-  void setupServiceNameHC() {
+  void setupServiceNameHC(const std::string& authority = "") {
```

Comment:
I think in here we prefer to not use a default for an arg, so probably `absl::nullopt` at the call site is "better".